### ISSUE TITLE
Could not view complete image in CropImageActivity

### ISSUE THREAD

**Issue by prasad456** [ArthurHub/Android-Image-Cropper#317]
Could not view complete Image in CropImageActivity, what I selected from camera or galley.
Please check below screenshot, which image was being out the image view.

![image](https://user-images.githubusercontent.com/6705718/27476413-4c92c7d8-5826-11e7-8081-52c14cf3ca75.png)

**Comment by prasad456** [comment]
I tested your sample which was downloaded. it works fine

after I integrated to my Project. this issue raised. Please let know If I may Miss anything

**Comment by ArthurHub** [comment]
can you paste the code you use

**Comment by prasad456** [comment]
MainActivity.class
// On Click
CropImage.activity(null)
                        .setGuidelines(CropImageView.Guidelines.ON)
                        .setAspectRatio(1,1)
                        .setCropShape(CropImageView.CropShape.OVAL)
                        .setFixAspectRatio(true)
                        .start(MainActivity.this);

//OnActivityResult(...){
if (requestCode == CropImage.CROP_IMAGE_ACTIVITY_REQUEST_CODE) {
                CropImage.ActivityResult result = CropImage.getActivityResult(data);
                if (resultCode == RESULT_OK) {
                    addGroupImage.setImageURI(result.getUri());
                } else if (resultCode == CropImage.CROP_IMAGE_ACTIVITY_RESULT_ERROR_CODE) {
                    customToast.show("Unable to get Image");
                }
            }
}

Issue was at the time of crop Selection.

**Comment by ArthurHub** [comment]
can you try without the theme

**Comment by prasad456** [comment]
I'm unable to view the toolbar

**Comment by ArthurHub** [comment]
right, but if the rest of the UI is correct I think it may be something with `Theme.AppCompat.Light` theme... maybe you have overridden something there...

**Comment by prasad456** [comment]
My Complete style.xml

`<style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
        
        <item name="colorPrimary">@color/colorPrimary</item>
        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
        <item name="colorAccent">@color/colorAccent</item>
        <item name="windowActionBar">true</item>
        <item name="windowNoTitle">true</item>
        <item name="android:windowLightStatusBar" tools:targetApi="23">true</item>
    </style>

    <style name="CropTheme" parent="Theme.AppCompat.Light">
        
        <item name="colorPrimary">@color/colorPrimary</item>
        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
        <item name="colorAccent">@color/colorAccent</item>
    </style>

    <style name="MyDialogTheme" parent="Theme.AppCompat.Light.Dialog.Alert">
        <item name="colorControlNormal">@color/colorPrimaryDark</item>
        <item name="colorControlActivated">@color/colorPrimaryDark</item>
        <item name="textColorAlertDialogListItem">@color/colorPrimaryDark</item>
        <item name="colorAccent">@color/colorPrimaryDark</item>
        <item name="android:textColorPrimary">@color/colorPrimaryDark</item>
    </style>

    <style name="RatingBar" parent="Theme.AppCompat">
        <item name="colorControlNormal">@color/dividerColor</item>
        <item name="colorControlActivated">@color/colorPrimaryDark</item>
    </style>`

**Comment by prasad456** [comment]
This style will be in above style.xml
<style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
        <item name="colorPrimary">@color/colorPrimary</item>
        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
        <item name="colorAccent">@color/colorAccent</item>
        <item name="windowActionBar">true</item>
        <item name="windowNoTitle">true</item>
        <item name="android:windowLightStatusBar" tools:targetApi="23">true</item>
    </style>

**Comment by ArthurHub** [comment]
I can't reproduce it...

**Comment by prasad456** [comment]
Can you tell me any recommended theme/style to use
I tried all it couldn't work. It Works for other projects.
May I missed anything in the library while integration?
